packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -633,34 +625,28 @@ Future<void> buildGradleAar({
   command.add(aarTask);
 
   final Stopwatch sw = Stopwatch()..start();
-  int exitCode = 1;
-
+  RunResult result;
   try {
-    exitCode = await processUtils.stream(
+    result = await processUtils.run(
       command,
       workingDirectory: project.android.hostAppGradleRoot.path,
       allowReentrantFlutter: true,
       environment: gradleEnv,
-      mapFunction: (String line) {
-        // Always print the full line in verbose mode.
-        if (logger.isVerbose) {
-          return line;
-        }
-        return null;
-      },
     );
   } finally {
     status.stop();
   }
   flutterUsage.sendTiming('build', 'gradle-aar', Duration(milliseconds: sw.elapsedMilliseconds));
 
-  if (exitCode != 0) {
-    throwToolExit('Gradle task $aarTask failed with exit code $exitCode', exitCode: exitCode);
+  if (result.exitCode != 0) {
+    printError(result.stderr, wrap: false);
```

Comment:
Could there be anything useful on stdout?

Reply:
Actually, it could be useful in some situations when understanding the project.